### DISCUSSION TITLE
Tui should allow creation of ACRs

### DISCUSSION THREAD

**Discussion by corriganjeff** [JanssenProject/jans#6533] [General]
The TUI does not allow for the creation of regular ACRs - this seems like a missing functionality. I can edit the other acrs but I can't de-SAMLize a test ACR I created. 

The other confusing thing is that the [docs](https://docs.jans.io/v1.0.20/admin/config-guide/default-authentication-method-config/) show how to export/update the default authentication method - which outputs:
{
  "defaultAcr": "string"
}

As seen in the screenshot, "string" is not the default acr - any ideas on why this differs from what's visible in the tui? 

![Screenshot 2023-11-09 at 10 10 33 AM](https://github.com/JanssenProject/jans/assets/36687287/709a21a0-5588-4d11-a2d5-afac561a978b)
![Screenshot 2023-11-09 at 10 10 53 AM](https://github.com/JanssenProject/jans/assets/36687287/36de7021-63aa-46a1-88d1-5f11c36c0341)

**Comment by ossdhaval** [comment]
CC: @devrimyatar for comments

**Comment by corriganjeff** [comment]
^bump on the creation of standard ACRs via TUI

**Reply by ossdhaval** [reply]
Looping in the concerned devs: @devrimyatar and may be @pujavs as well.

**Comment by devrimyatar** [comment]
Hello,

TUI does allow creation of ACR. For example I added a test ldap:
![ldap-test](https://github.com/JanssenProject/jans/assets/6398752/7e1cb47e-fba9-4245-9dc1-6319a7f750a4)

While adding this ldap, I checked *Default Authn Method* option
![myldap_edit](https://github.com/JanssenProject/jans/assets/6398752/6b46ad17-2f82-4a11-9bd9-4d5339a1c223)

As you can see in Authn list, *myldap* is default ACR (see *X* at the right of the row)
![authn_list](https://github.com/JanssenProject/jans/assets/6398752/7cd98fe5-99dc-48fb-a60c-fffbd6b5f00c)

Here is CLI query:
![cli_query](https://github.com/JanssenProject/jans/assets/6398752/9435258f-e168-4646-afd6-4739b0c57da8)

In the doc, this is just schema:
```
{
"defaultAcr": "string"
}
```
It means *defaultAcr* is a string propery, *string* is not the value.
